scripts/plopfile.js

```diff
@@ -66,9 +67,9 @@ module.exports = function (plop) {
         if (scopes.includes('@react-aria')) {
           actions.push({
             type: 'addMany',
-            templateFiles: 'plop-templates/@react-aria/**',
-            base: 'plop-templates/@react-aria/',
-            destination: `packages/@react-aria/${packageName}`,
+            templateFiles: '../plop-templates/@react-aria/**',
+            base: '../plop-templates/@react-aria/',
```

Comment:
just noticed all the forward slashes... does plop work in windows?

Reply:
good question... I can try setting up my personal computer to test it out (nothing but games on that one lol, I miss having the windows laptop in the office :()
Prob something to handle outside this PR tho

Reply:
I was able to create a new react-spectrum component on Windows. So generally, it seems to work.

Reply:
Sweeeet, thanks for testing that @mischnic